cves/CVE-2017-5052.yml

```
@@ -80,9 +81,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: There were automated unit tests in this vulnerability and the original
```

This should help fix the CI
```suggestion
  answer: |
    There were automated unit tests in this vulnerability and the original
    code was unit tested. The fix did not involve improving the automated tests. 
  code: |
    The bug was an extremely niche situation, so while the code was unit tested,
    it did not get the use case that caused the vulnerability.
    fix: A unit test was added for this issue. 
```